pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -86,6 +84,31 @@ public BrokerReduceService(PinotConfiguration config) {
     _reduceExecutorService = Executors.newFixedThreadPool(numThreadsInExecutorService, reduceThreadFactory);
   }
 
+  private static void updateAlias(QueryContext queryContext, BrokerResponseNative brokerResponseNative) {
```

Comment:
Let's not use IDE to rearrange the code (there is a checkbox during reformat). We don't usually follow that rule, and that is not really help the readability.

Reply:
done